docs/reference/modules/i18n.md

```diff
@@ -32,6 +32,7 @@ Each locale may have the following settings:
 | `label` | String | The human-readable label for the locale. |
 | `hostname` | String | A hostname (e.g., 'example.net') that will trigger the locale to be shown. |
 | `prefix` | String | A URL path prefix that will trigger the locale to be shown. |
+| `private` | Boolean | Setting this to true exposes this locale to loged-in users only. |
```

Comment:
logged-in

Reply:
Good catch. Sorry, not sure how I did that. Carefulness turned up to 11.